README.md

```diff
@@ -1,31 +1,42 @@
 # ethfs-cli
 
 ## Installation
-Globally:
+### Globally:
 ```bash
 npm install -g ethfs-cli
-ethfs-cli upload -f <directory|file> -a <address> -p <private-key> -r [rpc] -t [upload-type]
 ```
 
-Locally:
+Once installed, you can upload file or directory using:
+```
+ethfs-cli upload -f <directory|file> -a <address> -p <private-key> -c [chain-id] -t [upload-type]
+```
+
+### Locally:
 ```bash
 npm install ethfs-cli
-npx ethfs-cli upload -f <directory|file> -a <address> -p <private-key> -r [rpc] -t [upload-type]
+```
+
+After installation, use npx to run the command:
+```
+npx ethfs-cli upload -f <directory|file> -a <address> -p <private-key> -c [chain-id] -t [upload-type]
 ```
 <br/>
 
+
 ## Command
-| Short Name | Full Name                    | description                                |   
-|------------|------------------------------|--------------------------------------------|
-| -p         | --privateKey                 | private key                                |
-| -a         | --address                    | contract address / domain name             |
-| -f         | --file                       | upload file path / name                    |
-| -c         | --chainId                    | chain id                                   |
-| -r         | --rpc                        | provider url                               |
-| -t         | --type                       | file save type<br/>on chain: 1<br/>blob: 2 |
-| -g         | --gasPriceIncreasePercentage | gas price increase percentage              |
+| Short Name | Full Name        | description                                                                |   
+|------------|------------------|----------------------------------------------------------------------------|
+| -p         | --privateKey     | private key                                                                |
+| -a         | --address        | contract address / domain name                                             |
+| -f         | --file           | upload file path / name                                                    |
+| -c         | --chainId        | chain id                                                                   |
+| -r         | --rpc            | provider url                                                               |
+| -t         | --type           | file upload type:<br/>calldata: `1` or `calldata` <br/>blob: `2` or `blob` |
+| -g         | --gasIncPct      | gas price increase percentage                                              |
+| -s         | --threadPoolSize | number of threads for concurrent file uploads                              |
```

Comment:
What is the default value?

Reply:
[fix](https://github.com/ethstorage/ethfs-cli/pull/18#pullrequestreview-2250615060)